pkg/api/v1/register.go

```diff
@@ -23,8 +23,11 @@ import (
 	"k8s.io/kubernetes/pkg/runtime"
 )
 
+// GroupName is the group name use in this package
+const GroupName = api.GroupName
```

Comment:
I think what @lavalamp means is `const GroupName = ""` here, so that user can import v1 without importing the internal API (after we remove conversions).